pwndbg/commands/got.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pwndbg.chain
+import pwndbg.checksec
+import pwndbg.commands
+import pwndbg.enhance
+import pwndbg.file
+import pwndbg.which
+
+from pwndbg.color import green
+from pwndbg.color import light_yellow
+
+
+@pwndbg.commands.Command
+@pwndbg.commands.OnlyWhenRunning
+@pwndbg.commands.OnlyWithFile
+def got():
+    '''
+    Show the state of the Global Offset Table
+    '''
+
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+    cs_out = pwndbg.checksec.checksec(local_path)
+
+    file_out = pwndbg.file.file(local_path)
+    if "statically" in file_out:
+        return "Binary is statically linked."
+
+    jmpslots = pwndbg.elf.getjmpslots(local_path)
+    if not len(jmpslots):
+        return "NO JUMP_SLOT entries available in the GOT"
+
+    if cs_out['PIE']:
+        bin_text_base = pwndbg.memory.page_align(pwndbg.elf.entry())
+
+    relro_status = "No RELRO"
+    if cs_out['RELRO'] == 2:
+        relro_status = "Full RELRO"
```

Comment:
Do you think that it is not worth having a structured output from checksec results? Without that  in the commands code we will have to write `if "PIE enabled" in cs_out['PIE']` instead of `if cs_out['PIE']`. That's not a big problem, but the dependency from checksec output still remains... 

I agree that is not so good that we depends from checksec output, a solution could be to implement directly the techniques inside the script instead of using it as external tool, but again, is it worth it?